plugins/karate/KarateLight.cpp

```diff
@@ -253,6 +253,12 @@ bool KarateLight::ReadBack(uint8_t *rd_data, uint8_t *rd_len) {
 
   // read payload-data (if there is any)
   uint8_t payload_size = rd_buffer[CMD_HD_LEN];
+  if (payload_size > CMD_MAX_LENGTH - CMD_HD_LEN) {
```

Comment:
Not here - they only add noise. 

We really have to stop you add parens all over the place :)